src/NServiceBus.RavenDB.AcceptanceTests/App_Packages/NSB.AcceptanceTests.6.0.0-unstable1607/ScenarioDescriptors/AllTransports.cs

```diff
@@ -14,24 +14,7 @@ protected AllTransports()
             AddRange(ActiveTransports);
         }
 
-        static IEnumerable<RunDescriptor> ActiveTransports
-        {
-            get
-            {
-                if (activeTransports == null)
-                {
-                    //temporary fix until we can get rid of the "AllTransports" all together
-                    activeTransports = new List<RunDescriptor>
-                    {
-                        Transports.Default
-                    };
-                }
-
-                return activeTransports;
-            }
-        }
-
-        static ICollection<RunDescriptor> activeTransports;
+        static IEnumerable<RunDescriptor> ActiveTransports => new List<RunDescriptor> { Transports.Default };
```

Comment:
What does this comment provide? As long as it is not tracked in an issue
tracker by the original author it is "lost" anyway

On Monday, February 29, 2016, Colin Higgins <notifications@github.com>
wrote:

> In
> src/NServiceBus.RavenDB.AcceptanceTests/App_Packages/NSB.AcceptanceTests.6.0.0-unstable1607/ScenarioDescriptors/AllTransports.cs
> <https://github.com/Particular/NServiceBus.RavenDB/pull/174#discussion_r54435375>
> :
>
> > -            {
> > -                if (activeTransports == null)
> > -                {
> > -                    //temporary fix until we can get rid of the "AllTransports" all together
> > -                    activeTransports = new List<RunDescriptor>
> > -                    {
> > -                        Transports.Default
> > -                    };
> > -                }
> > -
> > -                return activeTransports;
> > -            }
> > -        }
> > -
> > -        static ICollection<RunDescriptor> activeTransports;
> > +        static IEnumerable<RunDescriptor> ActiveTransports => new List<RunDescriptor> { Transports.Default };
>
> Are we okay with losing the above comment? @danielmarbach
> <https://github.com/danielmarbach> @DavidBoike
> <https://github.com/DavidBoike>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus.RavenDB/pull/174/files#r54435375>
> .
>


-- 
*Don't miss the Async/Await Webinar Series*
*• Learn more <http://go.particular.net/l/27442/2016-02-15/681bjl> →*